test/sortWith.js

```diff
@@ -0,0 +1,89 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+var albums = [
```

Comment:
Nice test data! I wouldn't mind something easier to look at and play around with though. Just something like

```js
[
  {a:'x', b:1, c:100},
  {a:'x', b:2, c:100},
  {a:'x', b:3, c:100},
  {a:'x', b:1, c:200},
  {a:'x', b:1, c:300},
  {a:'y', b:1, c:100},
...
]
```

or what have you. Not a big deal though if everyone is ok as it is.